spacy/language.py

```diff
@@ -809,6 +810,14 @@ def from_disk(self, path, exclude=tuple(), disable=None):
             # Convert to list here in case exclude is (default) tuple
             exclude = list(exclude) + ["vocab"]
         util.from_disk(path, deserializers, exclude)
+
+        # download the KB for the entity linking component - requires the vocab
```

Comment:
This code feels a little arbitrary and lost here? Maybe we could add it as a deserializer above? Since it's a few lines of code, maybe the `kb` module could expose a deserializer helper function that does all of this and that follows the same format as the other deserializers?

I think it's cleaner to do it all in `util.from_disk`, and it'd also handle the exclusion etc. automatically, so we don't have to reimplement all of this here.

Reply:
Yea, I agree the current solution is not ideal ;-) Not sure yet how to rewrite though, the main problem being that the KB needs the vocab, which is only available after running `util.from_disk`, right?

Reply:
Yes, for sure. I turned to this ugliness because I somehow missed that the `deserializers` are in an `OrderedDict` so the `vocab` is available in time for the `kb` construction. Should be fixed in the latest commits.